samples/Playground/Playground.Shared/AppHost.cs

```diff
@@ -43,6 +43,7 @@ public static IApplicationBuilder ConfigureApp(this IApplicationBuilder builder)
 			.ConfigureServices((context, services) =>
 			{
 				services
+					.AddScoped<IThemeService, ThemeService>()
```

Comment:
This should be done in the `.UseThemeSwitching()` extension. Is there a reason for this to be scoped vs Singleton?

Reply:
Changed per Nick's request. I think it was something related with the dependencies being injected in the Service

Reply:
totally fine just wanted to understand the reason why we were using a scoped service there